### PR TITLE
Bumps + Improvements in file structure in production + fixes to config loading

### PR DESCRIPTION
**Pull request by benadida**
As title says :)

### CODE REVIEW

**Review by carolinemodic — APPROVED** [review]

**Comment by benadida** [comment]
confirmed that `election-manager` and `bmd` work as full installs on a VM with these changes.

File: setup-machine.sh [hunk 1]
```diff
@@ -45,10 +52,15 @@ sudo cp config/20auto-upgrades /etc/apt/apt.conf.d/
 # make sure machine never shuts down on idle, and does shut down on power key (no hibernate or anything.)
 sudo cp config/logind.conf /etc/systemd/
 
+# directory structure
+sudo mkdir /vx
+sudo mkdir /vx/code
+sudo mkdir -p /vx/data/module-scan
```
Comment (eventualbuddha) [inline setup-machine.sh:58]: Will this be writable by `vx-*`? I don't see a `chown` for it.

**Review by eventualbuddha — APPROVED** [review]

File: setup-machine.sh (hunk 1 above)
Comment (benadida) [reply setup-machine.sh:58]: in fact no, I just tried ballot-scanner and it failed :) Will fix.

Comment (benadida) [reply setup-machine.sh:58]: fixed!

**Pull request merged**